lib/Newsletter/NewslettersRepository.php

```diff
@@ -31,4 +40,66 @@ public function findActiveByTypes($types) {
       ->getQuery()
       ->getResult();
   }
+
+  public function getStandardNewsletterSentCount(DateTimeInterface $since): int {
+    return (int)$this->doctrineRepository->createQueryBuilder('n')
+      ->select('COUNT(n)')
+      ->join('n.queues', 'q')
+      ->join('q.task', 't')
+      ->andWhere('n.type = :type')
+      ->andWhere('n.status = :status')
+      ->andWhere('t.status = :taskStatus')
+      ->andWhere('t.processedAt >= :since')
+      ->setParameter('type', NewsletterEntity::TYPE_STANDARD)
+      ->setParameter('status', NewsletterEntity::STATUS_SENT)
+      ->setParameter('taskStatus', ScheduledTaskEntity::STATUS_COMPLETED)
+      ->setParameter('since', $since)
+      ->getQuery()
+      ->getSingleScalarResult() ?: 0;
+  }
+
+  public function getAnalytics(): array {
+    // for automatic emails join 'event' newsletter option to further group the counts
+    $eventOptionIdSubquery = $this->entityManager->createQueryBuilder()
+      ->select('nof.id')
+      ->from(NewsletterOptionFieldEntity::class, 'nof')
+      ->andWhere('nof.newsletterType = :eventOptionFieldType')
+      ->andWhere('nof.name = :eventOptionFieldName')
+      ->getQuery();
+
+    $results = $this->doctrineRepository->createQueryBuilder('n')
+      ->select('n.type, eventOption.value AS event, COUNT(n) AS cnt')
+      ->leftJoin('n.options', 'eventOption', Join::WITH, "eventOption.optionField = ({$eventOptionIdSubquery->getDQL()})")
```

Comment:
The subquery will be probably executed for every newsletter (not 100% sure) so maybe we could do 2 queries here.